tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -225,3 +225,20 @@ extension XCScheme.BuildableReference: Hashable {
         hasher.combine(blueprintName)
     }
 }
+
+extension XCScheme.BuildableReference {
+    var sortValue: String {
+        return [
+            buildableName,
+            blueprintName,
+            referencedContainer,
+            buildableIdentifier,
+        ].joined(separator: "|")
+    }
+}
+
+extension Sequence where Element == XCScheme.BuildableReference {
+    var inStableOrder: [XCScheme.BuildableReference] {
+        return sorted { $0.sortValue < $1.sortValue }
```

Comment:
`BlueprintName` is the target name, which is unique within a project (and we aren't doing workspaces), so we can use just that for sorting. We want to use `sortByLocalizedStandard()` for any user visible sorting:

```suggestion
        return sortByLocalizedStandard(\.blueprintName)
```